pvlib/temperature.py

```diff
@@ -821,3 +825,101 @@ def noct_sam(poa_global, temp_air, wind_speed, noct, module_efficiency,
     heat_loss = 1 - module_efficiency / tau_alpha
     wind_loss = 9.5 / (5.7 + 3.8 * wind_adj)
     return temp_air + cell_temp_init * heat_loss * wind_loss
+
+
+def prilliman(temp_cell, wind_speed, unit_mass=11.1, coefficients=None):
+    """
+    Smooth short-term cell temperature transients using the Prilliman model.
+
+    The Prilliman et al. model [1]_ applies a weighted moving average to
```

Comment:
I'm -1 on using the term "exponentially weighted" because that implies that the weight are of the form exp(k * delta_t) where k is a constant. In this model, k=k(t) is a function of t (P in Eq. 11 changes at each time step).